modules/monitoring-creating-a-monitoringstack-object-for-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// monitoring/cluster-observability-operator/configuring-the-cluster-observability-operator-to-monitor-a-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-a-monitoringstack-object-for-cluster-observability-operator_{context}"]
+= Creating a MonitoringStack object for Cluster Observability Operator
```

Comment:
```suggestion
= Creating a MonitoringStack object for the Cluster Observability Operator
```
Maybe consider adding an attribute for the Operator name to save you from typing it out every time?

Reply:
Fixed.

I'll also add an attribute, and be thankful for it later.